src/Core/SettingsChangesHistory.cpp

```diff
@@ -224,6 +224,7 @@ const VersionToSettingsChangesMap & getSettingsChangesHistory()
             {"backup_restore_keeper_max_retries_while_initializing", 0, 20, "New setting."},
             {"backup_restore_keeper_max_retries_while_handling_error", 0, 20, "New setting."},
             {"backup_restore_finish_timeout_after_error_sec", 0, 180, "New setting."},
+            {"use_concurrency_control", true, false, "Disable concurrency control by default to avoid unfair CPU distribution between SELECTs and INSERTS."},
```

Comment:
```suggestion
            {"use_concurrency_control", false, true, "Disable concurrency control by default to avoid unfair CPU distribution between SELECTs and INSERTS."},
```

Reply:
And move to 24.12.